### PR TITLE
VACMS-2860: Remove Old Date Field

### PR DESCRIPTION
**Pull request by kaise-lafrai**
## Description

THIS MUST BE HELD until front end team has removed it from vets-website (Merged 02/12)

See https://github.com/department-of-veterans-affairs/vets-website/pull/16015

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ethanteague** [comment]
Still seeing field_date in the graphql query for events in vets-web. This shouldn't be merged (yet).
image

**Comment by swirtSJW** [comment]
@kaise-lafrai   When you have a chance can you rebase this?  It has conflicts.

@ethanteague can you pair with Kaise and teach him your process for determining if this field has been removed from the front end?

@kaise-lafrai  If you and Ethan determine the frontend has fully removed it, lets get the ready for merge.

**Comment by ethanteague** [comment]
@kaise-lafrai @swirtSJW @kevwalsh - I rebased this pr, and updated vets-wesbite repo to remove / update all legacy graphql, template, and transformer instances in this pr: https://github.com/department-of-veterans-affairs/vets-website/pull/16015
We will need to wait for it to hit prod before we merge in 3516.

**Comment by ethanteague** [comment]
Moving back to in progress to fix failing phpunit test. Setting new date time field programmatically is a little more hinky than it was with original field.

File: docroot/modules/custom/va_gov_govdelivery/src/Service/ProcessStatusBulletin.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function processNode(NodeInterface $node) {
       // Pull the data from the situation update fields $this->situationUpdate.
       $template_variables['message'] = $this->situationUpdate->get('field_wysiwyg')->value;
       $template_variables['situation_update'] = TRUE;
-      $time = $this->situationUpdate->get('field_date_and_time')->date->getTimestamp();
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_govdelivery/src/Service/ProcessStatusBulletin.php:79]: I'll need to test this.

**Review by swirtSJW — APPROVED** [review]
Tested this out on devshop and confirmed no change of date behavior.

**Comment by swirtSJW** [comment]
image

**Pull request merged**